src/ring_theory/localization.lean

```diff
@@ -902,43 +963,22 @@ variables {R K}
 
 lemma to_map_eq_zero_iff [comm_ring K] (φ : fraction_map R K) {x : R} :
   x = 0 ↔ φ.to_map x = 0 :=
-begin
```

Comment:
Hmm, this name seems backwards... Would you mind fixing it (even though it wasn't part of your PR...)
I suggest leaving the name, but swapping the two sides of the statement.